app/models/assignment.rb

```diff
@@ -914,6 +915,28 @@ def has_peer_review_assignment?
     not pr_assignment.nil?
   end
 
+  def create_peer_review_assignment_if_not_exist
+    if has_peer_review and Assignment.where(parent_assignment_id: id).empty?
+      peerreview_assignment = Assignment.new
+      peerreview_assignment.parent_assignment = self
+      peerreview_assignment.submission_rule = NoLateSubmissionRule.new
+      peerreview_assignment.assignment_stat = AssignmentStat.new
```

Comment:
We do (they determine whether there can be late submissions and mark statistics). You've set the correct defaults here.